### ISSUE TITLE
networks are not automatically removed

### ISSUE THREAD

**Issue by ftardif** [testcontainers/moby-ryuk#2]
I am running a docker compose with testcontainers. But I do not explicitly stop the DockerComposeContainer at the end of my test (because the container is statically reused by multiple test in the test suite). I was expecting the moby-ryuk to delete the containers and the networks at the end. It actually does delete the container correctly, but it does not seem to delete the networks.

```
~/devel/docker-fonse$ docker ps
CONTAINER ID        IMAGE                               COMMAND                  CREATED             STATUS                  PORTS                          NAMES
8c7a7503accb        confluentinc/cp-zookeeper:5.1.0     "/etc/confluent/dock…"   1 second ago        Up Less than a second   2181/tcp, 2888/tcp, 3888/tcp   z2wsbcee3xqm_zookeeper_1
0a1b8c2ba32f        docker/compose:1.8.0                "/usr/bin/docker-com…"   3 seconds ago       Up 2 seconds                                           quizzical_jennings
686039115d69        quay.io/testcontainers/ryuk:0.2.2   "/app"                   8 seconds ago       Up 7 seconds            0.0.0.0:32827->8080/tcp        testcontainers-ryuk-fcc1e2e8-88a6-448b-98f3-510af7731e0a
~/devel/docker-fonse$ docker logs testcontainers-ryuk-fcc1e2e8-88a6-448b-98f3-510af7731e0a -f
2019/01/08 18:42:54 Starting on port 8080...
2019/01/08 18:42:54 Connected
2019/01/08 18:42:54 Adding {"label":{"org.testcontainers.sessionId=fcc1e2e8-88a6-448b-98f3-510af7731e0a":true,"org.testcontainers=true":true}}
2019/01/08 18:42:54 Adding {"label":{"com.docker.compose.project=z2wsbcee3xqm":true}}
2019/01/08 18:43:26 EOF
2019/01/08 18:43:26 Disconnected
2019/01/08 18:43:36 Timed out waiting for connection
2019/01/08 18:43:36 Deleting {"label":{"org.testcontainers.sessionId=fcc1e2e8-88a6-448b-98f3-510af7731e0a":true,"org.testcontainers=true":true}}
2019/01/08 18:43:36 Deleting {"label":{"com.docker.compose.project=z2wsbcee3xqm":true}}
2019/01/08 18:43:38 Removed 6 container(s), 0 network(s), 0 volume(s)
~/devel/docker-fonse$ docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
~/devel/docker-fonse$ docker network list
NETWORK ID          NAME                  DRIVER              SCOPE
12cb5757d5e4        bridge                bridge              local
c7829c02e770        host                  host                local
7b2bb170377d        none                  null                local
3b836122fd7c        z2wsbcee3xqm_patate   bridge              local
~/devel/docker-fonse$ 

```

Is this the expected behavior? I was under the impression that the ryuk was meant exactly for this purpose...

**Comment by bsideup** [comment]
Reported as https://github.com/docker/compose/issues/6636

**Comment by slnowak** [comment]
@bsideup any updates on that? I think I've just encountered this issue and the corresponding docker compose ticket is closed

**Comment by rnorth** [comment]
I'm looking into this at the moment; I suspect the most straightforward path is to upgrade the version of docker-compose that we use to at least support v2.1 compose files. 

Will have a try with that approach first.

**Comment by rnorth** [comment]
This should have been resolved in #1847, with an upgrade to the version of docker compose in use. Release notes here: https://github.com/testcontainers/testcontainers-java/releases/tag/1.12.4
